docs/quick-start/single-binary-three.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 2
+---
+
+# Three-node cluster with a single binary
+
+First, download a prebuilt copy of TigerBeetle.
+
+On macOS/Linux:
+
+```console
+git clone https://github.com/tigerbeetle/tigerbeetle; ./tigerbeetle/bootstrap.sh
+```
+
+On Windows:
+
+```console
+git clone https://github.com/tigerbeetle/tigerbeetle; .\tigerbeetle\bootstrap.ps1
+```
+
+Want to build from source locally? Add `-build` as an argument to the bootstrap script.
+
```

Comment:
This feels a bit odd. Three-node-cluster feels like a "real" thing, not just a quick-start, so it probably should recomend "production" way of getting tigerbeetle, and not our bootstrap script (the assumption here is that `git clone https://github.com/tigerbeetle/tigerbeetle; ./tigerbeetle/bootstrap.sh` is _not_ how we recommend folks to get TB in prod). 

So I expect to see here something like "Download tigerbeetle binary", where Download is a link to the docs which explains all the ways to get tigerbetele (with the primary one being a github release).

Reply:
Added install page in https://github.com/tigerbeetle/tigerbeetle/pull/1321, hat tip to jorangreef for suggesting that we just add a markdown table with releases to our docs!